llvm-bazel/llvm-project-overlay/llvm/BUILD

```diff
@@ -2049,16 +2049,26 @@ cc_library(
     ],
 )
 
+# There is some deep layering violation between these libraries and JITLink.
+# To workaround this, the same headers are included in multiple rules.
+cc_library(
+    name = "Orc_layering_violation_headers",
```

Comment:
It's probably good to keep naming consistent, even if this is a special case.

Reply:
Yeah the naming in this file is a bit all over the place, but at least right now, weird workaround rules that don't correspond to any CMake libraries do have underscores. We could come up with a more consistent methodology for that. I kind of like verbose and ugly names for these kinds of verbose and ugly workarounds though :grin: